packages/docmanager-extension/src/index.tsx

```diff
@@ -612,11 +611,7 @@ function addCommands(
       return false;
     }
     const context = docManager.contextForWidget(currentWidget);
-    return !!(
-      context &&
-      context.contentsModel &&
-      context.contentsModel.writable
-    );
+    return !!context?.writable;
```

Comment:
Changed not needed because the `writable` state of the content model is _corrected_ in `Context._updateContentsModel`:

```suggestion
    return !!context?.contentsModel?.writable;
```